Source/WebCore/accessibility/AccessibilityNodeObject.cpp

```diff
@@ -607,7 +608,7 @@ void AccessibilityNodeObject::addChildren()
         return;
 
     // The only time we add children from the DOM tree to a node with a renderer is when it's a canvas.
-    if (renderer() && !node->hasTagName(canvasTag))
+    if (renderer() && WebCore::elementName(node.get()) != ElementName::HTML_canvas)
```

Comment:
Could use the `Node&` version of `elementName` (the previous code dereferenced it unconditionally)